docs/en/setup/service-agent/java-agent/Supported-list.md

```diff
@@ -148,6 +148,7 @@ metrics based on the tracing data.
   * [Apache Commons DBCP](https://github.com/apache/commons-dbcp) 2.x
   * [Alibaba Druid](https://github.com/alibaba/druid) 1.x
   * [HikariCP](https://github.com/brettwooldridge/HikariCP) 3.x -> 4.x
+  * [C3P0](https://github.com/swaldman/c3p0) 0.9.0 -> 0.10.0
```

Comment:
I think you should update meter plugin list as well. I noticed, it seems other `Connection Pool` plugins are not in the list as well, could you check?

Reply:
Same page here, https://github.com/apache/skywalking-java/blob/main/docs/en/setup/service-agent/java-agent/Supported-list.md#meter-plugins

Reply:
ok